### PR TITLE
Test SP6

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by dcermak** [comment]
I'm afraid this won't start working properly until we publish the SLE_BCI repo (should be at public beta).

**Comment by dirkmueller** [comment]
> I'm afraid this won't start working properly until we publish the SLE_BCI repo (should be at public beta).

We have published the bci repo this week (in agreement with release management)

**Comment by dcermak** [comment]
This will need https://github.com/SUSE/BCI-dockerfile-generator/pull/822 plus probably additional fixes

**Comment by dirkmueller** [comment]
This is now ready. the remaining issue is a bug in the java packaging that is being addressed.

File: tests/test_all.py [hunk 1]
```diff
@@ -310,6 +309,10 @@ def test_no_orphaned_packages(container_per_test: ContainerData) -> None:
         "sles-release",
         "ALP-dummy-release",
     }
+    # bsc#1219115
+    if OS_VERSION == "15.6":
+        known_orphaned_packages.union({'java-11-openjdk', 'java-11-openjdk-headless'})
```
Comment (dcermak) [inline tests/test_all.py:314]: `union` does not modify in-place
```suggestion
        known_orphaned_packages = known_orphaned_packages.union({'java-11-openjdk', 'java-11-openjdk-headless'})
```

**Comment by dcermak** [comment]
LGTM, but the sp6 repo is totally busted atm and needs a refresh. I'd like to merge this once the sp6 repo pipeline is at least working a little bit, so that we have a somewhat stable CI pipeline.

**Pull request merged**